browser-test/src/profile_stability.test.ts

```diff
@@ -1,32 +1,37 @@
-import {test, expect} from '@playwright/test'
-import {createTestContext, isHermeticTestEnvironment} from './support'
+import {test, expect} from './support/civiform_fixtures'
+import {isHermeticTestEnvironment} from './support'
 import {BASE_URL, FROZEN_PLAY_SESSION_COOKIE_VALUE} from './support/config'
 
-test.describe('user HTTP sessions', () => {
-  const ctx = createTestContext()
-
-  interface Profile {
-    id: string
-    roles: string[]
-    clientName: string
-  }
-
-  // This test ensures that an existing user profile, stored in a browser cookie,
-  // is recognized and properly deserialized by the server.
-  //
-  // This guards against changes that unexpectedly affect serialization.
-  test('recognizes the profile from a frozen cookie', async () => {
-    // Play encrypts cookies with the server secret:
-    // https://www.playframework.com/documentation/2.8.x/ApplicationSecret
-    //
-    // For this test to run as a prober, we would need to provision one frozen
-    // cookie value per environment and pass that in. That is more trouble than
-    // it is worth, since the hermetic test (which relies on using the default
-    // server secret) will detect any breaking changes to profile serialization.
+test.describe(
+  'User HTTP sessions',
+  {tag: ['@uses-fixtures', '@parallel-candidate']},
```

Comment:
Just ignore `@parallel-candidate`. Trying to tag things that may potentially be able to be run in parallel some day. But don't count your chickens.